client/blocks/app-banner/index.jsx

```diff
@@ -65,6 +66,15 @@ class AppBanner extends Component {
 		userAgent: ( typeof window !== 'undefined' ) ? navigator.userAgent : '',
 	};
 
+	preventNotificationsClose( appBanner ) {
+		const appBannerNode = ReactDom.findDOMNode( appBanner );
+		if ( ! appBannerNode ) {
+			return;
+		}
+		appBannerNode.addEventListener( 'mousedown', event => event.stopPropagation(), false );
+		appBannerNode.addEventListener( 'touchstart', event => event.stopPropagation(), false );
+	}
```

Comment:
👍 I've updated to remove the listeners. We still need the findDomNode call since the ref callback is not returning a bare dom node.